js/market.js

```diff
@@ -1,59 +1,71 @@
 
 $.get('http://localhost:5000/plans')
     .done((result) => {
-        // let product=JSON.stringify(result);
-        let f = document.getElementById('form-box');
+        console.log(result);
+        let f = document.getElementsByTagName('div')[2];
         let counter = 1;
-        for (let i in result.data) {
+        for (let i in result) {
 
             let div = document.createElement('div');
             div.className = 'columns';
             let ul = document.createElement('ul');
             ul.className = 'price';
             let li = document.createElement('li');
             li.className = 'header';
-            li.innerHTML = result.data[i].name;
+            li.innerHTML = result[i].name;
             ul.appendChild(li);
 
-            if (!result.data[i].prices[1]) {
+            if (!result[i].price_year) {

```

Comment:
keep the convention- either use camelCased names or snake_case do not mix both. on the server side you've used camelCased names